tests/unit/cli/test_pyinstaller_imports.py

```diff
@@ -22,3 +22,42 @@ def test_hook_not_contain_self_defined_command_packages(self):
 
         for name in command_package_names:
             self.assertNotIn(name, hidden_imports.SAM_CLI_HIDDEN_IMPORTS)
+
+    def test_hook_contain_all_samcli_hook_packages(self):
```

Comment:
How is this test passing? I understand the imports are done because of `walk_modules`, but how are the modules present within SAM_CLI_HIDDEN_IMPORTS

Reply:
@sriram-mv it's the logic here: SAM_CLI_HIDDEN_IMPORTS = list(samcli_modules) + [...other imports]

Reply:
Every walked module will be in the list samcli_modules